src/engraving/libmscore/vibrato.cpp

```diff
@@ -22,6 +22,8 @@
 
 #include <cmath>
 
+#include <QGuiApplication>
```

Comment:
what is this for?

maybe we can include necessary missing types directly?

Reply:
this is for `qApp->translation`
In the future should be replaced to use our `trc` function